charts/kyuubi/values.yaml

```diff
@@ -101,17 +101,22 @@ server:
 kyuubiConfDir: /opt/kyuubi/conf
 kyuubiConf:
   # The value (templated string) is used for kyuubi-env.sh file
-  # See https://kyuubi.apache.org/docs/latest/deployment/settings.html#environments for more details
+  # See https://kyuubi.readthedocs.io/en/master/deployment/settings.html#environments for more details
   kyuubiEnv: ~
 
   # The value (templated string) is used for kyuubi-defaults.conf file
-  # See https://kyuubi.apache.org/docs/latest/deployment/settings.html#kyuubi-configurations for more details
-  kyuubiDefaults: ~
+  # See https://kyuubi.readthedocs.io/en/master/deployment/settings.html#kyuubi-configurations for more details
+  kyuubiDefaults: |
+
 
   # The value (templated string) is used for log4j2.xml file
-  # See https://kyuubi.apache.org/docs/latest/deployment/settings.html#logging for more details
+  # See https://kyuubi.readthedocs.io/en/master/deployment/settings.html#logging for more details
   log4j2: ~
 
+  extraFiles: {}
```

Comment:
it's valuable to change in a separate PR. let's focus on one thing in each PR, to make the future visitor easy to understand what we were doing.

Reply:
Thanks! got it!

Reply:
what is the use case for this property?

Reply:
Users may use Kyuubi/Spark w/ Hive MetaStore, then `hive-site.xml` is required

Reply:
And another use case is `FileSessionConfAdvisor` introduced in #3742, and emm... it isn't mentioned in docs, and we appreciate contributors who can help us to improvement the docs 😄

Reply:
@pan3793  Would it be better to add specific properties instead? I.e. `hiveSite` etc

Reply:
> @pan3793 Would it be better to add specific properties instead? I.e. `hiveSite` etc

I agree to use specific properties for specific files, but there are some non-enumerable files, e.g. in `FileSessionConfAdvisor` use case, there may have

- `kyuubi-session-etl.conf`
- `kyuubi-session-ad-hoc.conf`
- `kyuubi-session-admin.conf`
- `kyuubi-session-<...>.conf`